geozarr-spec.md

```diff
@@ -75,6 +75,13 @@ The following standard names are recommended to describe coordinates variables f
 
 The **grid_mapping** CF variable defined by DataArray variable defines  the coordinate reference system (CRS) used for the horizontal spatial coordinate values. The grid_mapping value indicates the Auxliary variable that holds all the CF attribute describing the CRS. 
 
+In GeoZarr, the grid_mapping variable can contain a **GeoTransform** attribute. A GeoTransform is a vector of six double precision digits: [X_offset, X0, X1, Y_offset, Y0, Y1] `X_offset` and `Y_offset` are the grid origin. `X0` and `Y0` are the grid spacing per column. `X1` and `Y1` are the grid spacing per row. If `X0 = Y1 = 0`, the grid is axis aligned. In GeoZarr, coordinates are always always aligned with a cell center and it is left to a data user to decide if values are constant across a cell or valid only at the precise cell cetner. The following equations can be used to create derive georeferenced cell centers. 
```

Comment:
"In GeoZarr, coordinates ... cetner" (cetner -> center); does this sentence refer to the case where the coordinates are a Zarr array? If yes, it may be worth mentioning that CF conventions leave this open (if bounds are specfied, coordinate array values do not have to be halfway between the bounds); if no, then I would leave this out as the equations below specify how row/col space maps to x/y coordinates. The equations below map _continuous_ col/row values to x/y; see e.g. [here](https://gdal.org/api/gdaldataset_cpp.html#_CPPv4N11GDALDataset15GetGeoTransformEPd), in particular the section _In a north up image, padfTransform[1] is the pixel width, and padfTransform[5] is the pixel height. The upper left corner of the upper left pixel is at position (padfTransform[0],padfTransform[3])._ I would reuse that section and write: _In a north up image, `X0` is the pixel width, and `Y1` is the pixel height. The upper left corner of the upper left pixel is at position (`X_offset`, `Y_offset`). The coordinates of the cell _center_ of row i, column j are obtained by using j+0.5 for `column` and i+0.5 for `row` in the following equations: _

(Also: "create derive" -> derive)

Reply:
Updated text. I think this better aligns with what's in the wild now.

Reply:
Yes, you may want to add that row & column start counting at 0.